com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -10,11 +12,37 @@ namespace Unity.Netcode
     /// </summary>
     [AddComponentMenu("Netcode/" + nameof(NetworkObject), -99)]
     [DisallowMultipleComponent]
-    public sealed class NetworkObject : MonoBehaviour
+
+    public sealed class NetworkObject : MonoBehaviour, IInterestObject<NetworkObject>
     {
         [HideInInspector]
         [SerializeField]
-        internal uint GlobalObjectIdHash;
+        public uint GlobalObjectIdHash;
```

Comment:
Is this something we actually need to expose? I mean this would not be public to devs

Reply:
Apologies, left over from an experiment